docs/additional-examples/evm-swap.md

```diff
@@ -0,0 +1,656 @@
+# Cross Chain EVM Swap
+In this section, we will demonstrate how you can use PKPs and Lit Actions to perform a cross-chain swap. There are three parts to this flow:
+
+- Understanding Lit PKPs as escrow
+- Lit Action
+- Implementing Mint/Grant/Burn with SDK
+
+## 1) Understanding Lit PKPs as escrow
+
+A Private key is meant for signing transactions which are executed on the blockchain, a network verifies if the transaction is a legitimate one and then executes it. With Lit Network we can generate a special type of private keys which can be programmed to sign transactions only when certain conditions are met. With this, we can create several escrow mechanisms, and we will look into how we can execute a cross-chain swap.
```

Comment:
i think we can abstract away some of these details here, they feel a bit excessive. I think we can just briefly introduce PKPs (like we do here https://developer.litprotocol.com/user-wallets/pkps/overview in the overview) and then say that for this use case we are going to use them as escrow accounts to support the cross chain swap flow

Reply:
This was meant to give an overview of how we can "generally" use pkps for escrow-like cases and not only limit to defining pkps. As for bridging, messaging and swapping, the only thing that'll change will be the lit action.

I tried to make it use friendly but if it feels overwhelming, I can reduce it.

Reply:
yea imo i think we can simplify things a bit. I think we should start by briefly introducing PKPs generally before honing in the specific use case of escrow

Reply:
check below
https://github.com/LIT-Protocol/docs/pull/330/commits/30185e8fdd79532766306cf2736f33c5726e9f50#diff-0e36021eb2e9b13aa34e50ec77474d75d50bab8d18cba35c954841b81d9efee7R10